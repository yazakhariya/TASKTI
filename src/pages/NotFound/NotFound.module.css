.wrapper {
  min-height: 100vh;
  width: 100%;
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  background-color: #ffe192;
}
