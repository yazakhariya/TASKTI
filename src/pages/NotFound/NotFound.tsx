import loader from 'src/assets/loader.svg'
import styles from './NotFound.module.css'

export default function NotFound() {
  return (
    <div className={styles.wrapper}>
      <h2>Cтраница не найдена...</h2>
      <img width="100px" src={loader} />
      <p>
        <a href="/">Visit Homepage</a>
      </p>
    </div>
  )
}
