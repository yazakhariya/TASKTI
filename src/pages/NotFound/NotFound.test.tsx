import { render, screen } from '@testing-library/react'
import { describe, expect, it } from 'vitest'
import NotFound from './NotFound'

describe('NotFound', () => {
  it('should render properly', () => {
    render(<NotFound />)
    expect(screen.getByTestId('my-test-id')).toBeDefined();
  })
})