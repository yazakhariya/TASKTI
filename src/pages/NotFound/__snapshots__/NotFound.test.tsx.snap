// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`NotFound > should match snapshot 1`] = `
<h2
  data-testid="my-test-id"
>
  Cтраница не найдена...
</h2>
`;
